app/models/queries/FormQueries.py

```diff
@@ -0,0 +1,52 @@
+from app.models import Forms
+from app.models import Files
+
+
+
+class FormQueries():
```

Comment:
Would you turn this class into just functions, we want all the Query helper functions to be functions and not classes, so that there is not need to instantiate an object for the function.
Can you please make that change